### PR TITLE
Support for kw MAXVALUE/MINVALUE.

### PR DESCRIPTION
**Pull request by osae**
Added functionality, and some simple testing.

### CODE REVIEW

**Comment by joakim-hove** [comment]
Short comment: If you rebase on screaming latest and greatest master you will see that the code for handling the keyword input box was changed in a recent PR.

File: lib/eclipse/EclipseState/Eclipse3DProperties.cpp [hunk 1]
```diff
@@ -858,6 +862,34 @@ namespace Opm {
     }
 
 
+    void Eclipse3DProperties::handleMAXVALUEKeyword( const DeckKeyword& deckKeyword, BoxManager& boxManager) {
```
Comment (joakim-hove) [inline lib/eclipse/EclipseState/Eclipse3DProperties.cpp]: Please add a brief comment that the `MAXVALUE` and `MINVALUE` keywords are processed in place for the current value of the keyword, and do not "stick" as an attribute of the keyword.

**Comment by joakim-hove** [comment]
OK - thank you for adding a test. This will be merged when it has been rebased (i.e. the box handling has been updated to the approach used in current master).

**Comment by osae** [comment]
Thanks for comments. The issues have now hopefully been sorted out.

**Comment by joakim-hove** [comment]
jenkins build this with downstreams please

**Pull request merged**